client/src/components/modals_and_popovers/StatelessModal.tsx

```diff
@@ -15,14 +15,18 @@ interface StatelessModalProps {
   body: React.ReactNode;
   footer?: React.ReactNode;
   close_text?: string;
-  include_close_button_in_header: boolean;
+  include_close_button_in_header?: boolean;
   additional_dialog_class_name?: string;
-}
+};
 
 export class StatelessModal extends React.Component<StatelessModalProps> {
   static defaultProps = {
+    subtitle: "",
```

Comment:
I think because this has a default of `""`, even though it's declared optional in the explicit types, it's going to internally say that it's type is _just_ `string`. That means in practice we're giving the empty string the special meaning of "do not render a subtitle". I think it probably makes sense to not render any subtitle related DOM when the string is empty, but for the sake of the component's interface it's clearer if it explicitly takes `undefined` to mean "render no subtitle". Same on the other optional props here.

So, to be clear, I think they should just be optional string props, and should not be in default props.